core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/RebindManagerImpl.java

```diff
@@ -92,6 +92,9 @@
     public static final ConfigKey<RebindFailureMode> REBIND_FAILURE_MODE =
             ConfigKeys.newConfigKey(RebindFailureMode.class, "rebind.failureMode.rebind",
                     "Action to take if a failure occurs during rebind", RebindFailureMode.FAIL_AT_END);
+    public static final ConfigKey<RebindFailureMode> ADD_CONFIG_FAILURE_MODE =
+            ConfigKeys.newConfigKey(RebindFailureMode.class, "rebind.failureMode.addConfig",
+                    "Action to take if a failure occurs when setting a config value. It could happen coercion of the value type to fail.", RebindFailureMode.FAIL_AT_END);
```

Comment:
I just preserved it like the behavior it was before this change. Startup was failing the same way.
Svet, do you really want me to change the default to CONTINUE ?